Experimental/DistributedAuthoritySample/Assets/Scripts/Player/ThirdPersonRBController.cs

```diff
@@ -0,0 +1,227 @@
+using System;
```

Comment:
For the purposes of testing, and before we integrate the locomotion + animations, this can go through, but let's move this to the Temp/ folder inside of Scripts.

Reply:
How can I move the script to Temp without it being ignored by github?

Reply:
In Rider, if you navigate to the file itself and stage it, you'll get a prompt to confirm adding an ignored file by Git.
Through the command line you can do `git add -f <path>` to force stage the file.